go/libraries/doltcore/env/environment.go

```diff
@@ -428,16 +428,25 @@ func (dEnv *DoltEnv) getRPCCreds() (credentials.PerRPCCredentials, error) {
 			panic(err)
 		}
 
-		dCreds, err := creds.JWKCredsReadFromFile(dEnv.FS, filepath.Join(dir, kid+".jwk"))
+		c, err := creds.JWKCredsReadFromFile(dEnv.FS, filepath.Join(dir, kid+".jwk"))
+		return c, c.IsPrivKeyValid() && c.IsPubKeyValid(), err
+	}
 
-		if err != nil {
-			return nil, ErrInvalidCredsFile
-		}
+	return creds.EmptyCreds, false, nil
+}
 
-		return dCreds, nil
+func (dEnv *DoltEnv) getRPCCreds() (credentials.PerRPCCredentials, error) {
+	dCreds, valid, err := dEnv.UserRPCCreds()
+
+	if err != nil {
+		return nil, ErrInvalidCredsFile
 	}
 
-	return nil, nil
+	if !valid {
```

Comment:
If it's not valid, shouldn't it be an ErrInvalidCredsFile?

Reply:
Good call. Fixed!

Reply:
No, this was right. The `!valid` case is really the empty / missing case.